source/includes/_account.md

```diff
@@ -5598,7 +5598,7 @@ None.
 
 **Returns**
 
-If successful, returns a status code of 201 and an empty body.
+If successful, returns a status code of 201 and a list of Tag objects.
```

Comment:
Could you double check this is true (same question with the other endpoints below)?

https://github.com/killbill/killbill/blob/cbec994cc841d9f55c1f56e73093153b06760523/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/JaxRsResourceBase.java#L230

In Java, if `followLocation` is set, the client will call back Kill Bill to fetch the tags: https://github.com/killbill/killbill-client-java/blob/853b03236e10f24ab06bbb9065175d72c6c1ddce/src/main/java/org/killbill/billing/client/RequestOptions.java#L63

If it's false, it should return an empty body (just like the cURL).

FYI for historical context: https://github.com/killbill/killbill/issues/917

Reply:
Fixes to specify location header instead of empty body